index.html

```diff
@@ -12,9 +12,10 @@
     <div class="gradient-border" id="camera">
       <video id="video">Video stream not available.</video>
     </div>
+    <div class="loader"></div>
     <div id="subtitle">
-    </div>
+    </div>  
```

Comment:
```suggestion
    </div>
```

Reply:
what to do here?

Reply:
there was 2 unnecessary whitespace after div. remove it